htdocs/user/card.php

```diff
@@ -1203,7 +1205,9 @@
 	{
 		print '<tr><td>'.$langs->trans("AccountancyCode").'</td>';
 		print '<td>';
-		print '<input size="30" type="text" name="accountancy_code" value="'.dol_escape_htmltag(GETPOST('accountancy_code', 'alphanohtml')).'">';
+		$accountancy_code = (GETPOSTISSET('accountancy_code') ? GETPOST('accountancy_code', 'alpha') : $conf->global->SALARIES_ACCOUNTING_ACCOUNT_PAYMENT);
+		print $formaccounting->select_account($accountancy_code, 'accountancy_code', 1, null, 1, 1, '');
```

Comment:
Note that a lot of users does not use the accounting double entry. 
They just enter a value to have it into the CSV file when making export for the bookkeeper.
In such a case, we must be able to enter a free text.
So you should introduce a test if $conf->accounting->enabled to use the combo list (otherwise the list is just empty).